boot/pod/test/boot/file_test.clj

```diff
@@ -22,7 +22,7 @@
 
 (deftest relative-to-test
   (testing "Nil base"
-    (is (= "out/goog/base.js"          (str (relative-to nil      (io/file "out/goog/base.js"))))))
+    (is (thrown? AssertionError        (str (relative-to nil      (io/file "out/goog/base.js"))))))
```

Comment:
Yeah... `nil` is covered by a `:pre` condition in `relative-to`.  I thought about removing the test, but wasn't sure if there was a good reason for it to be there.  I think it's the only other option though, unless the `:pre` shouldn't be there.